stubs/ibm-db/ibm_db.pyi

```diff
@@ -0,0 +1,233 @@
+WCHAR_NO: int
+WCHAR_YES: int
+
+@final
+class IBM_DBClientInfo:
+    def __new__(cls: type[Self], *args: object, **kwargs: object) -> Self: ...
+    APPL_CODEPAGE: str
+    CONN_CODEPAGE: str
+    DATA_SOURCE_NAME: str
+    DRIVER_NAME: str
+    DRIVER_ODBC_VER: str
+    DRIVER_VER: str
+    ODBC_SQL_CONFORMANCE: str
+    ODBC_VER: str
+
+@final
+class IBM_DBConnection:
+    def __new__(cls: type[Self], *args: object, **kwargs: object) -> Self: ...
+
+@final
+class IBM_DBServerInfo:
+    def __new__(cls: type[Self], *args: object, **kwargs: object) -> Self: ...
+    DBMS_NAME: str
+    DBMS_VER: str
+    DB_CODEPAGE: str
+    DB_NAME: str
+    DFT_ISOLATION: str
+    IDENTIFIER_QUOTE_CHAR: str
+    INST_NAME: str
+    ISOLATION_OPTION: str
+    KEYWORDS: str
+    LIKE_ESCAPE_CLAUSE: bool
+    MAX_COL_NAME_LEN: bytes
+    MAX_IDENTIFIER_LEN: str
+    MAX_INDEX_SIZE: bytes
+    MAX_PROC_NAME_LEN: bytes
+    MAX_ROW_SIZE: bytes
+    MAX_SCHEMA_NAME_LEN: bytes
+    MAX_STATEMENT_LEN: bytes
+    MAX_TABLE_NAME_LEN: bytes
+    NON_NULLABLE_COLUMNS: str
+    PROCEDURES: bool
+    SPECIAL_CHARS: str
+    SQL_CONFORMANCE: str
+
+@final
+class IBM_DBStatement:
+    def __new__(cls: type[Self], *args: object, **kwargs: object) -> Self: ...
+
+def active(connection: IBM_DBConnection) -> bool: ...
+def autocommit(connection: IBM_DBConnection, value: int | None = ...) -> int | bool: ...
+def bind_param(
+    stmt: IBM_DBStatement,
+    parameter_number: int,
+    variable: str,
+    parameter_type: int | None = ...,
+    data_type: int | None = ...,
+    precision: int | None = ...,
+    scale: int | None = ...,
+    size: int | None = ...,
+) -> bool: ...
+def callproc(
+    connection: IBM_DBConnection, procname: str, parameters: tuple[object, ...]
+) -> IBM_DBStatement | tuple[object, ...] | None: ...
+def check_function_support() -> bool: ...
+def client_info(connection: IBM_DBConnection) -> IBM_DBClientInfo | bool: ...
+def close(connection: IBM_DBConnection) -> bool: ...
+def column_privileges(
+    connection: IBM_DBConnection,
+    qualifier: str | None = ...,
+    schema: str | None = ...,
+    table_name: str | None = ...,
+    column_name: str | None = ...,
+) -> IBM_DBStatement: ...
+def columns(
+    connection: IBM_DBConnection,
+    qualifier: str | None = ...,
+    schema: str | None = ...,
+    table_name: str | None = ...,
+    column_name: str | None = ...,
+) -> IBM_DBStatement: ...
+def commit(connection: IBM_DBConnection) -> bool: ...
+def conn_error(connection: IBM_DBConnection | None = ...) -> str: ...
+def conn_errormsg(connection: IBM_DBConnection | None = ...) -> str: ...
+def conn_warn(connection: IBM_DBConnection | None = ...) -> str: ...
+def connect(
+    database: str, user: str, password: str, options: dict[int, int | str] | None = ..., replace_quoted_literal: int = ...
+) -> IBM_DBConnection | None: ...
+def createdb(connection: IBM_DBConnection, dbName: str, codeSet: str | None = ..., mode: str | None = ...) -> bool: ...
```

Comment:
```suggestion
def createdb(connection: IBM_DBConnection, dbName: str, codeSet: str = ..., mode: str = ...) -> bool: ...
```
It stringifies the arguments before it checks for None, I don't think None really works.